codegenerator/cli/npm/envio/package.json.tmpl

```diff
@@ -28,6 +28,9 @@
     "envio-darwin-x64": "${version}",
     "envio-darwin-arm64": "${version}"
   },
+  "dependencies": {
+    "rescript": "11.1.3"
+  },
```

Comment:
Had to add the dependency because of the error https://github.com/enviodev/hyperindex/actions/runs/11438088469/job/31820281579

Reply:
Is this still needed after adding file protocol?